pkg/cmd/dev/util.go

```diff
@@ -136,6 +137,29 @@ func addCommonTestFlags(cmd *cobra.Command) {
 	cmd.Flags().Bool(shortFlag, false, "run only short tests")
 }
 
+var workdir string
+
+func pathInWorkdir(ctx context.Context, ex *exec.Exec, subPath string) (string, error) {
```

Comment:
We already have the `getWorkspace()` method for this so this isn't necessary, but either way `dev doctor` already has the path to the workspace.

Reply:
Works for me. You can wrap the heuristics with `if !isTesting {}` and it shouldn't require any updates to the unit tests in that case.

Reply:
I went ahead and updated the tests too